src/js/index.js

```diff
@@ -88,3 +88,15 @@ document.addEventListener('DOMContentLoaded', () => {
 
   getConfig();
 });
+
```

Comment:
This came back

Reply:
Thanks, thought I caught all the bad merges